### DISCUSSION TITLE
Discontinuation

### DISCUSSION THREAD

**Discussion by ludo550** [community/community#108137] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Shelex, Good Sir, 
I am trying to use this reporter in our framework but was recently told that you have an intention to discontinue it's maintenance. Could you confirm the accuracy of this statement?

**Comment by mw-xyz** [comment]
Which reporter?

**Comment by AlanGreene** [comment]
@ludo550 It looks like you posted this in the wrong place. I see you also posted to https://github.com/Shelex/cypress-allure-plugin/discussions/226 which seems like the intended repo.

Please close [this discussion](https://github.com/orgs/community/discussions/108137).

**Comment by ludo550** [comment]
Done @AlanGreene . Sorry for the inconvenience
